Task.js

```diff
@@ -133,6 +134,8 @@ class Task extends Unit {
       isSimulation: false,
       isAssesment: false,
       assets_path: assets_path,
+      lab_data: lab_data,
+      exp_info: exp_info,
```

Comment:
Why do we need lab_data, exp_info and page_plugins in page_data?

Reply:
I needed them intially for other purposes,
right now I just need page_plugins, so I can pass attributes (like questions) at build time.